src/site/filters/liquid.js

```diff
@@ -51,4 +54,10 @@ module.exports = function registerFilters() {
     });
     return JSON.stringify(facilityList);
   };
+
+  liquid.filters.widgetFacilityDetail = facility => {
+    const facilityLocatorApiId = facility.split('_')[1].toUpperCase();
+    const id = `vha_`.concat(facilityLocatorApiId).replace(/"/g, '');
+    return JSON.stringify(id);
```

Comment:
you may not need the `JSON.stringify(id)` and can just return the `id` from the line above (or just return that line)

Reply:
the stringify is needed but the string replace was not.